vhost-user-backend/src/backend.rs

```diff
@@ -87,6 +89,18 @@ pub trait VhostUserBackend: Send + Sync {
     /// function.
     fn set_backend_req_fd(&self, _backend: Backend) {}
 
+    /// This function gets an owned file descriptor that the front-end can use otherwise
+    /// no file descriptor.
+    ///
+    /// A default implementation is provided as we cannot expect all backends to implement this
```

Comment:
I'd update this comment like we did for PR #267, to make clear that backends need to implement it if the feature is supported, otherwise we just return an error.